src/main/java/online/shop/model/Order.java

```diff
@@ -0,0 +1,38 @@
+package online.shop.model;
+
+import java.util.List;
+
+public class Order {
+    private Long id;
+    private List<Product> products;
+    private Long usedId;
+
+    public Order(List<Product> products, Long usedId) {
+        this.products = products;
```

Comment:
I think out mentors have been commenting on this saying that we will not receive a List of products as a parameter, and we should just initialize the List<Product> products in the constructor.